### PR TITLE
Merged Jabber and XMPP into "XMPP/Jabber"

### PR DESCRIPTION
**Pull request by Nyco**
Resolves Issue #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -282,10 +278,11 @@ These open source projects will help you to build a bot in order to automate you
 
 * [Hubot](https://hubot.github.com/) + [adapter](https://github.com/KasperDeng/Hubot-WeChat) (CoffeeScript, Node.js)
 
-### XMPP
+### XMPP/Jabber
```
Comment (exAspArk) [inline README.md:281]: Could you please change the names of the sections in the `Bots` list?

Comment (Nyco) [reply README.md:281]: Oops... Sorry.

Like this? Thanks!

Comment (exAspArk) [reply README.md:281]: Thanks! And also need to remove `Jabber` and update the anchor link 😸

Comment (exAspArk) [reply README.md:281]: I can do it actually if you don't mind :)

**Comment by exAspArk** [comment]
Thank you for your contribution! 🎉

**Pull request merged**